lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -286,6 +287,27 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL
+   * Collapsed content is opened automatically
+   * Element is highlighted via CSS selector
+   */
+  _jumpToAudit() {
+    const hash = window.location.hash;
+    if (hash) {
+      const audit = document.querySelector(hash);
+      if (audit) {
+        if (audit.firstElementChild) {
+          audit.firstElementChild.setAttribute('open', 'true');
+        }
+        if (audit.parentElement) {
+          audit.parentElement.setAttribute('open', 'true');
```

Comment:
in cases like accessibility we'll need to go up a couple levels :/ 

maybe replace this with a quick loop? untested example below :)
 
```js
for (let node = audit.parentElement; node; node = node.parentElement) {
  if (node.tagName === 'DETAILS') node.setAttribtue('open', 'true');
}

Reply:
we probably also don't want to go anywhere above `lh-container` either. The Lighthouse report might end up embedded in someone else's `<details>` :)

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
Yeah, `closest()` worked fine in my last project